source.go

```diff
@@ -35,7 +35,7 @@ type Source struct {
 
 	iterator  source.Iterator
 	config    source.Config
-	connPool  *pgxpool.Pool
+	conn      *pgxpool.Pool
```

Comment:
If you don't like `connPool` I'd rather use just `pool` than `conn`, reading the code and seeing `conn` can give the wrong impression IMO.